src/chainstate/burn/operations/stack_stx.rs

```diff
@@ -318,7 +320,7 @@ impl StackStxOp {
                 self.num_cycles, POX_MAX_NUM_CYCLES
             );
         }
-
+        increment_stack_stx_op();
```

Comment:
This should be somewhere in the top-level `Burnchain::sync()` method, such as right after when a block gets processed.  There's no guarantee anywhere that this method will be called only once.